### PR TITLE
Flush save indicators per message

### PR DESCRIPTION
**Pull request by elayabharath**
+ Subtler indicator instead of a warning sign
+ Remove unnecessary warning during the transition state

### CODE REVIEW

File: webapp/lib/app/configurator/standard_messages/controller.dart [hunk 1]
```diff
@@ -202,8 +202,7 @@ class MessagesConfiguratorController extends ConfiguratorController {
       await Future.wait([platform.updateSuggestedReplies(diffData.editedMessages), platform.deleteSuggestedReplies(diffData.deletedMessages)]);
       _view
         ..unsavedChanges = false
-        ..showSaveStatus('Saved!', autoHide: true)
-        ..clearUnsavedIndicators();
+        ..showSaveStatus('Published for saving!', autoHide: true);
```
Comment (marianamarasoiu) [inline webapp/lib/app/configurator/standard_messages/controller.dart:205]: Can we roll this back to "Saved!"? (even though we know it's not quite true)

"published for saving" won't make sense as a description to the users

File: webapp/lib/app/configurator/standard_messages/controller_messages_helper.dart [hunk 2]
```diff
@@ -200,24 +201,25 @@ class StandardMessagesManager {
...
-      } else if (localMessage.categoryName != storageMessage.categoryName) {
+      } else if (localMessage.categoryName != storageMessage.categoryName && localMessage.categoryName != standardMessage.categoryName) {
         // the category has been updated
         _view.categoriesById[standardMessage.categoryId].showAlternative(standardMessage.categoryName);
-      } else if (localMessage.groupName != storageMessage.groupName) {
+      } else if (localMessage.groupName != storageMessage.groupName && localMessage.groupName != standardMessage.groupName) {
         // the group has been updated
          _view.categoriesById[standardMessage.categoryId].groupsById[standardMessage.groupId].showAlternative(standardMessage.groupName);
-      } else if (localMessage.text != storageMessage.text || localMessage.translation != storageMessage.translation) {
+      } else if ((localMessage.text != storageMessage.text && localMessage.text != standardMessage.text) || (localMessage.translation != storageMessage.translation && localMessage.translation != standardMessage.translation)) {
```
Comment (marianamarasoiu) [inline webapp/lib/app/configurator/standard_messages/controller_messages_helper.dart:210]: let's split this on two lines to it reads a bit easier
```suggestion
      } else if ((localMessage.text != storageMessage.text && localMessage.text != standardMessage.text)
             || (localMessage.translation != storageMessage.translation && localMessage.translation != standardMessage.translation)) {
```

**Review by marianamarasoiu — APPROVED** [review]
thanks EB, LGTM with a couple of nits!

File: webapp/lib/app/configurator/standard_messages/controller.dart (hunk 1 above)
Comment (elayabharath) [reply webapp/lib/app/configurator/standard_messages/controller.dart]: Fixed!

File: webapp/lib/app/configurator/standard_messages/controller_messages_helper.dart (hunk 2 above)
Comment (elayabharath) [reply webapp/lib/app/configurator/standard_messages/controller_messages_helper.dart]: Fixed!

**Pull request merged**